h/static/scripts/directive/excerpt.coffee

```diff
@@ -0,0 +1,35 @@
+###*
+# @ngdoc directive
+# @name excerpt
+# @restrict AE
+# @description Checks to see if text is overflowing its container.
+# If so, it prepends/appends expanders/collapsers. Note, to work
+# the element needs a max height.
+###
+module.exports = ->
+  link: (scope, elem, attr, ctrl) ->
+    if scope.enabled == undefined
```

Comment:
It would be more idiomatic (and safer) to check `if typeof scope.enabled == 'undefined'`.